### PR TITLE
Deprecate legacy ConnectorSplitManager methods

### PR DESCRIPTION
**Pull request by electrum**

### CODE REVIEW

File: presto-accumulo/src/main/java/io/prestosql/plugin/accumulo/AccumuloSplitManager.java [hunk 1]
```diff
@@ -52,7 +53,7 @@ public AccumuloSplitManager(AccumuloClient client)
     }
 
     @Override
-    public ConnectorSplitSource getSplits(ConnectorTransactionHandle transactionHandle, ConnectorSession session, ConnectorTableHandle tableHandle, SplitSchedulingStrategy splitSchedulingStrategy)
+    public ConnectorSplitSource getSplits(ConnectorTransactionHandle transactionHandle, ConnectorSession session, ConnectorTableHandle tableHandle, SplitSchedulingStrategy splitSchedulingStrategy, DynamicFilter dynamicFilter)
```
Comment (findepi) [inline presto-accumulo/src/main/java/io/prestosql/plugin/accumulo/AccumuloSplitManager.java:56]: split the line

File: presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcSplitManager.java [hunk 2]
```diff
@@ -35,7 +36,7 @@ public JdbcSplitManager(JdbcClient jdbcClient)
     }
 
     @Override
-    public ConnectorSplitSource getSplits(ConnectorTransactionHandle transaction, ConnectorSession session, ConnectorTableHandle table, SplitSchedulingStrategy splitSchedulingStrategy)
+    public ConnectorSplitSource getSplits(ConnectorTransactionHandle transaction, ConnectorSession session, ConnectorTableHandle table, SplitSchedulingStrategy splitSchedulingStrategy, DynamicFilter dynamicFilter)
```
Comment (findepi) [inline presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcSplitManager.java:39]: split the line (others too)

Comment (findepi) [inline presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcSplitManager.java:39]: split the line

File: presto-accumulo/src/main/java/io/prestosql/plugin/accumulo/AccumuloSplitManager.java (hunk 1 above)
Comment (electrum) [reply presto-accumulo/src/main/java/io/prestosql/plugin/accumulo/AccumuloSplitManager.java:56]: I wanted to avoid making unrelated formatting changes

**Pull request merged**